### ISSUE TITLE
Open closed logs with grep highlighting

### ISSUE THREAD

**Issue by yooksi** [krasa/GrepConsole#188]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like to know if there is a way to open an existing (not open in stream) log file with grep and filter according to log levels. If not I would like to request this feature as IntelliJ lacks a good log viewer (or at least I am not aware of one).

**Describe alternatives you've considered**
There is a [comment](https://stackoverflow.com/questions/20804756/intellij-idea-how-to-view-txt-files-with-the-features-of-console-stack-trace#comment31248500_20807479) on Stackoverflow that points to this feature on Grep console but I was not able to find it.

**Additional context**
Post on Stackoverflow that provides more information about the request:
https://stackoverflow.com/questions/20804756/intellij-idea-how-to-view-txt-files-with-the-features-of-console-stack-trace

**Comment by krasa** [comment]
![image](https://user-images.githubusercontent.com/1160875/107789106-79e8a380-6d51-11eb-8906-6373c1b97897.png)

**Comment by krasa** [comment]
I usually drag the file into this - but you need to enable showing of the whole toolbar
![image](https://user-images.githubusercontent.com/1160875/107789310-b7e5c780-6d51-11eb-9da2-f513f1953e9f.png)

**Comment by yooksi** [comment]
I am aware of the Tail feature, which we can also do this by right clicking on the log file and `Open in -> Tail in Console`.
However this doesn't give us the ability to filter lines by log level like this:

![2021-02-12_185331](https://user-images.githubusercontent.com/16436920/107804777-a78b1800-6d64-11eb-8a99-fe140828aaf1.png)

I am not sure if this feature comes from Grep or IntelliJ, but it seems almost essential to working with logs. In the screen above I am switching between the console and log file output which both have the same content, but the log file (lower panel in the screenshot above) tab has the feature to filter according to log level which appears in the form of a drop down menu that can be seen on the right side.

This feature can only be accessed as the log is being streamed and is not present when opening tail in console. It also has an annoying but that sometimes does not print anything to console if it is not **focused** the moment the stream starts piping, but other then that works as expected. I would like to see the same but only for log files that are not open in a stream.

**Comment by krasa** [comment]
this thing does it 
![image](https://user-images.githubusercontent.com/1160875/107816518-2d16c400-6d75-11eb-8d51-4eea243d1660.png)

**Comment by yooksi** [comment]
Yes, I know how to configure the logs so they are used by those panels. 
I was just wondering if that feature for filtering by log level comes from **IntelliJ IDEA** or the **Grep plugin**.

**Comment by krasa** [comment]
from IntelliJ IDEA

**Comment by krasa** [comment]
> It also has an annoying but that sometimes does not print anything to console if it is not **focused**

https://youtrack.jetbrains.com/issue/IDEA-217617
